packages/react-scripts/template/src/Toast.css

```diff
@@ -0,0 +1,39 @@
+.Toast {
```

Comment:
The idea is to use this as a template for the user to customize it. So I think it's okay to leave it not namespaced. CMIIW